src/main/java/com/datadoghq/sketch/ddsketch/store/Store.java

```diff
@@ -71,6 +71,9 @@ default void mergeWith(Store store) {
      */
     Store copy();
 
+
+    void clear();
```

Comment:
yes, and I would also find this surprising as a user. Perhaps we could call this method `zero()` and reserve `clear()` for actually clearing the sketch?

Reply:
I think it's okay to use `clear()`, especially since it has the same behavior as the implementations of `Collection.clear()` or `Map.clear()` that I checked. I just think it's worth highlighting that behavior in the javadoc, or even make the fact that it's expected to reuse already allocated memory part of its contract.